crates/polars-plan/src/logical_plan/options.rs

```diff
@@ -278,6 +282,7 @@ impl ProjectionOptions {
         Self {
             run_parallel: self.run_parallel & other.run_parallel,
             duplicate_check: self.duplicate_check & other.duplicate_check,
+            should_broadcast: self.should_broadcast | other.should_broadcast,
```

Comment:
*question*: is this the correct semantics? Perhaps it should also be `&`.

Reply:
The conservative case is always broadcasting.

When do we merge these again?

Reply:
In `optimizer/cluster_with_columns.rs` are the only usages I can find. But not sure what that is doing.